src/test/skript/tests/regressions/4664-formatted time.sk

```diff
@@ -0,0 +1,20 @@
+test "formatted time":
+	set {_default} to "yyyy-MM-dd HH:mm:ss z"
+	set {_now} to now
```

Comment:
While unlikely, it is possible that this effect is executed in another second (e.g. at 2.999 seconds) and the checks below in the next second (e.g. at 3.001 seconds). It'd be better to use `now` directly, instead of a variable

Reply:
Or maybe even better use the variable in all places?

Reply:
> Or maybe even better use the variable in all places?

I was wondering about this since I couldn't see a reason why the expression needed re-using.

Reply:
That'd work too, up to you